src/commands.c

```diff
@@ -966,17 +1001,100 @@ int yh_com_get_pubkey(yubihsm_context *ctx, Argument *argv, cmd_format fmt) {
   return 0;
 }
 
+int yh_com_get_device_pubkey(yubihsm_context *ctx, Argument *argv,
```

Comment:
please add the function header comments all other functions have, brief description and what (if any) arguments the function has.

Reply:
Fixed

Reply:
See commit 23c464a8f8c92445ad4b85c271abafac5eb6e0ce